pkgs/by-name/st/stylelint-lsp/package.nix

```
@@ -41,11 +42,9 @@ stdenvNoCC.mkDerivation (finalAttrs: {
 
     mkdir -p $out/{bin,lib/${finalAttrs.pname}}
     mv {dist,node_modules} $out/lib/${finalAttrs.pname}
-    echo "
-      #!${lib.getExe bash}
-      ${lib.getExe nodejs} $out/lib/${finalAttrs.pname}/dist/index.js \$@
-    " > $out/bin/stylelint-lsp
-    chmod +x $out/bin/stylelint-lsp
+    chmod a+x $out/lib/${finalAttrs.pname}/dist/index.js
+    makeWrapper $out/lib/${finalAttrs.pname}/dist/index.js $out/bin/stylelint-lsp  \
+      --prefix PATH : "${lib.getExe nodejs}"
```

Sorry to bother again with this, but I found an even simpler solution. By adding `nodejs` to `buildInputs` (and removing it from `nativeBuildInputs` as it is redundant), the shebang will be patched automatically in the fixup phase, so manually wrapping the program is not necessary.

```suggestion
    ln -s $out/lib/${finalAttrs.pname}/dist/index.js $out/bin/stylelint-lsp
```

---

Thank you for guiding me! :)
Updated as suggested and tested.